src/components/accordion/accordion.tsx

```diff
@@ -140,4 +167,12 @@ export class Accordion {
 
   private sortItems = (items: any[]): any[] =>
     items.sort((a, b) => a.position - b.position).map((a) => a.item);
+
+  private passPropsToAccordionItems = (): void => {
+    (
+      Array.from(
```

Comment:
Can we use the existing `this.items` instead of querying for the child items again?

Reply:
This makes sense as long as `this.items` gets updated when `accordion-item`s get added or removed from the light dom.  Is this currently handled in any way?

Reply:
Yeah good point the internal registration events are kind of an outdated pattern and wouldn't respond - an observer that responds to addition / removal of children and updates `this.items` is probably better. There's a handful of our ... more senior 👴👵 ... components that could probably benefit from a change like that.